boards/arm/nucleo_f446re/doc/nucleof446re.rst

```diff
@@ -0,0 +1,216 @@
+.. _nucleo_f446re_board:
+
+ST Nucleo F446RE
+################
+
+Overview
+********
+
+The Nucleo F446RE board features an ARM Cortex-M4 based STM32F446RE MCU
+with a wide range of connectivity support and configurations. Here are
+some highlights of the Nucleo F446RE board:
+
+
+- STM32 microcontroller in QFP64 package
+- Two types of extension resources:
+       - Arduino Uno V3 connectivity
```

Comment:
need a blank line before a (sub) list like this.  Also don't indent it so far or the doc processor treats this as a block quote.  The ``-`` should be aligned with the text of the outdented list above it:

    - Two types of extension resources:

      - Arduino Uno V3 Connectivity
      - ST morpho ...